config/app.default.php

```diff
@@ -244,6 +249,8 @@
              * which is the recommended value in production environments
              */
             //'init' => ['SET GLOBAL innodb_stats_on_metadata = 0'],
+            
+            'url' => env('DATABASE_URL', ''),
```

Comment:
This PR is however is also meant to allow parsing of .env files like @josegonzalez's Plugin does, isn't it?
Ignore my concerns if you deem them to be irrelevant, anything further can be explained on IRC and is a question on my side then.